conda-store-server/conda_store_server/server/app.py

```diff
@@ -335,6 +336,10 @@ def start(self):
 
         # start worker if in standalone mode
         if self.standalone:
+            address = "localhost" if self.address == "0.0.0.0" else self.address
+            print(
+                f"Starting standalone conda-store server at http://{address}:{self.port}"
+            )
```

Comment:
I don't like this. If we're using `self.address` we should print `self.address`. I understand this is trying to be helpful, but it'll only make issues harder to debug later. I'd suggest to remove this entirely.